test/storage/file.js

```diff
@@ -1025,6 +1025,22 @@ describe('File', function() {
     });
   });
 
+  describe('makePublic', function() {
+    it('should make the file public', function(done) {
+
+      // If this mocked method (THAT DOES NOTHING!) is removed, nothing works.
+      file.acl.makeAclObject_ = function() {};
```

Comment:
This is just how I see effective unit testing working for our library, not specific to this situation. `file.acl.makeAclObject_` is being called because when you call `file.makePublic`, it in turn calls `file.acl.add`, which calls `file.acl.makeAclObject_`. Instead of letting it fall through all those places, we should interrupt it on its way to `file.acl.add`.